### PR TITLE
9). Add topic files 11 to 14

### PR DESCRIPTION
**Pull request by jameshkramer**
@mairaw
This PR is for user story #1576037.

It adds four topic files, makes corresponding deletions from the C# and VB folders, and adds redirection entries.

cc: @tfosmark

Contributes to #4728

### CODE REVIEW

**Comment by pkulikov** [comment]
@jameshkramer should this and #17021 target the `linq-to-xml` branch, not `master`?

**Comment by jameshkramer** [comment]
@pkulikov, yes, sorry. I'll fix.

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -1249,6 +1249,21 @@
       "redirect_url": "/dotnet/standard/linq/build-linq-xml-examples",
       "redirect_document_id": true
     },
+    {
+      "source_path": "docs/csharp/programming-guide/concepts/linq/creating-xml-trees-linq-to-xml-2.md",
+      "redirect_url": "/dotnet/standard/linq/create-xml-trees-csharp",
```
Comment (mairaw) [inline .openpublishing.redirection.json]: I don't think you need the language in the file name. Let's keep it simple unless you need to distinguish it from another article

```suggestion
      "redirect_url": "/dotnet/standard/linq/create-xml-trees",
```

File: .openpublishing.redirection.json [hunk 2]
```diff
@@ -4244,6 +4259,18 @@
...
+      "source_path": "docs/visual-basic/programming-guide/concepts/linq/functional-construction-linq-to-xml.md",
+      "redirect_url": "/dotnet/standard/linq/functional-construction"
+    },
+    {
+      "source_path": "docs/visual-basic/programming-guide/concepts/linq/how-to-parse-a-string.md",
+      "redirect_url": "/dotnet/standard/linq/parse-string"
+    },
+    {
+      "source_path": "docs/visual-basic/programming-guide/concepts/linq/introduction-to-xml-literals.md",
+      "redirect_url": "/dotnet/standard/linq/xml-literals-visual-basic"
```
Comment (mairaw) [inline .openpublishing.redirection.json]: same here 

```suggestion
      "redirect_url": "/dotnet/standard/linq/xml-literals"
```

File: docs/standard/linq/create-xml-trees-csharp.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+For information about using the results of LINQ queries as the content for an <xref:System.Xml.Linq.XElement>, see [Functional construction](functional-construction.md).
+
+## Example: Construct elements
+
+The signatures of the <xref:System.Xml.Linq.XElement> and <xref:System.Xml.Linq.XAttribute> constructors let you pass the contents of the element or attribute as arguments to the constructor. Because one of the constructors takes a variable number of arguments, you can pass any number of child elements. Of course, each of those child elements can contain their own child elements. For any element, you can add any number of attributes.
+
+When adding <xref:System.Xml.Linq.XNode> (including <xref:System.Xml.Linq.XElement>) or <xref:System.Xml.Linq.XAttribute> objects, if the new content has no parent, the objects are simply attached to the XML tree. If the new content already is parented, and is part of another XML tree, the new content is cloned, and the newly cloned content is attached to the XML tree. The last example in this article demonstrates this.
+
+To create a `contacts`<xref:System.Xml.Linq.XElement>, you could use the following code:
```
Comment (mairaw) [inline docs/standard/linq/create-xml-trees-csharp.md]:
```suggestion
To create a `contacts` <xref:System.Xml.Linq.XElement>, you could use the following code:
```

File: docs/standard/linq/create-xml-trees-csharp.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+ms.assetid: cc74234a-0bac-4327-9c8c-5a2ead15b595
+---
+
+# Create XML trees in C# (LINQ to XML)
+
+This article provides information about creating XML trees in C#.
+
+For information about using the results of LINQ queries as the content for an <xref:System.Xml.Linq.XElement>, see [Functional construction](functional-construction.md).
+
+## Example: Construct elements
```
Comment (mairaw) [inline docs/standard/linq/create-xml-trees-csharp.md]:
```suggestion
## Construct elements
```

File: docs/standard/linq/parse-string.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+ms.assetid: 81e5686c-9658-42d8-a7e3-b11be0a2c98b
+---
+
+# How to parse a string
+
+This article shows how to create an XML tree using your choice of C# or Visual Basic.
+
+## Example: Create an XML tree
+
+he following example creates an XML tree. The C# version of the example uses the `XElement.Parse` method to parse an XML string to create the tree. This technique is also available in Visual Basic, but in this example the Visual Basic version uses XML literals.
```
Comment (mairaw) [inline docs/standard/linq/parse-string.md]:
```suggestion
The following example creates an XML tree. The C# version of the example uses the <xref:System.Xml.Linq.XElement.Parse%2A?displayProperty=nameWithType> method to parse an XML string to create the tree. This technique is also available in Visual Basic, but in this example, the Visual Basic version uses XML literals.
```

File: docs/standard/linq/parse-string.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 81e5686c-9658-42d8-a7e3-b11be0a2c98b
+---
+
+# How to parse a string
+
+This article shows how to create an XML tree using your choice of C# or Visual Basic.
```
Comment (mairaw) [inline docs/standard/linq/parse-string.md]:
```suggestion
This article shows how to create an XML tree using C# or Visual Basic.
```

File: docs/standard/linq/parse-string.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+            <City>Mercer Island</City>
+            <State>WA</State>
+            <Postal>68042</Postal>
+            </Address>
+            <NetWorth>11</NetWorth>
+        </Contact>
+    </Contacts>");
+Console.WriteLine(contacts);
+```
+
```
Comment (mairaw) [inline docs/standard/linq/parse-string.md]: wondering if we should make the article look more like what we had before:
https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2008/bb387054(v=vs.90)

File: docs/standard/linq/parse-string.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+            <Postal>68042</Postal>
+            </Address>
+            <NetWorth>11</NetWorth>
+        </Contact>
+    </Contacts>
+```
+
+The root `Contacts` node has two `Contact` nodes. To access some specific data in your parsed XML, use the [XElement.Elements()](xref:System.Xml.Linq.XContainer.Elements) method, which in this case returns the child elements of the root `Contacts` node. The following C# example prints the first `Contact` node to the console:
+
+```csharp
```
Comment (mairaw) [inline docs/standard/linq/parse-string.md]: Can you also add the VB version?

Here's the code:

```vb
Dim contactNodes As List(Of XElement) = contacts.Elements("Contact").ToList()
Console.WriteLine(contactNodes(0))
```

File: docs/standard/linq/parse-string.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+            <City>Mercer Island</City>
+            <State>WA</State>
+            <Postal>68042</Postal>
+            </Address>
+            <NetWorth>11</NetWorth>
+        </Contact>
+    </Contacts>
+```
+
+The root `Contacts` node has two `Contact` nodes. To access some specific data in your parsed XML, use the [XElement.Elements()](xref:System.Xml.Linq.XContainer.Elements) method, which in this case returns the child elements of the root `Contacts` node. The following C# example prints the first `Contact` node to the console:
```
Comment (mairaw) [inline docs/standard/linq/parse-string.md]:
```suggestion
The root `Contacts` node has two `Contact` nodes. To access some specific data in your parsed XML, use the [XElement.Elements()](xref:System.Xml.Linq.XContainer.Elements) method, which in this case returns the child elements of the root `Contacts` node. The following example prints the first `Contact` node to the console:
```

File: docs/standard/linq/parse-string.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
+---
+title: "How to parse a string"
+ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 81e5686c-9658-42d8-a7e3-b11be0a2c98b
+---
+
+# How to parse a string
```
Comment (mairaw) [inline docs/standard/linq/parse-string.md]:
```suggestion
# How to parse a string (LINQ to XML)
```

File: docs/standard/linq/parse-string.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
+---
+title: "How to parse a string"
```
Comment (mairaw) [inline docs/standard/linq/parse-string.md]: I'd add the suffix here too even though the original didn't have it

```suggestion
title: "How to parse a string - LINQ to XML"
```

File: docs/standard/linq/xml-literals-visual-basic.md [hunk 12]
```diff
@@ -0,0 +1,204 @@
+---
```
Comment (mairaw) [inline docs/standard/linq/xml-literals-visual-basic.md]: This is not connected to the TOC but I think if you rename the file as I'm suggesting to you, that would fix the problem

File: docs/standard/linq/xml-literals-visual-basic.md [hunk 13]
```diff
@@ -0,0 +1,204 @@
...
+title: XML Literals in Visual Basic
```
Comment (mairaw) [inline docs/standard/linq/xml-literals-visual-basic.md]:
```suggestion
title: XML Literals in Visual Basic - LINQ to XML
```

File: docs/standard/linq/xml-literals-visual-basic.md [hunk 14]
```diff
@@ -0,0 +1,204 @@
...
+ms.date: 07/20/2015
+ms.assetid: cc74234a-0bac-4327-9c8c-5a2ead15b595
+---
+
+# XML Literals in Visual Basic
```
Comment (mairaw) [inline docs/standard/linq/xml-literals-visual-basic.md]:
```suggestion
# XML Literals in Visual Basic (LINQ to XML)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @jameshkramer. I've left you some comments to address before we can merge, Let me know if you have any questions or if you need any help.

File: docs/standard/linq/parse-string.md (hunk 7 above)
Comment (jameshkramer) [reply docs/standard/linq/parse-string.md]: OK. Now it looks almost identical except for some additional material from the current C# version of the article.

File: docs/standard/linq/parse-string.md (hunk 8 above)
Comment (jameshkramer) [reply docs/standard/linq/parse-string.md]: Yes, I'm adding it.

**Review by mairaw — APPROVED** [review]
Looks great. Thanks @jameshkramer!

**Pull request merged**